plugin/storage/cassandra/schema.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+CQLSH_HOST=${CQLSH_HOST:-"cassandra"}
+CASSANDRA_WAIT_TIMEOUT=${CASSANDRA_WAIT_TIMEOUT:-"60"}
+DATACENTER=${DATACENTER:-"openshift"}
```

Comment:
this doesn't look like a good default. I am actually not sure what datacenter the official Cassandra image mentions.